deep_qa/data/instances/tuple_inference_instance.py

```diff
@@ -219,7 +222,10 @@ def read_from_line(cls, line: str, default_label: int=None):
         later.
         """
         fields = line.split("\t")
-        if len(fields) == 4:
+        if len(fields) == 5:
+            question_text, index, answers_string, background_string, label = fields
```

Comment:
The convention is to always have the index be the first field, so if you're adding a new one, it should go after the index.